dist/main.js

```diff
@@ -0,0 +1,35 @@
+let currTeam;
+$("#submit").click(function() {
+  let teamName = $("input").val();
+  $.get(`/teams/${teamName}`, function(response) {
+    renderer(response);
+    currTeam = response;
+  });
+});
+
+$("#dream").click(function() {
+  $.get("/dreamTeam", function(response) {
+    renderer(response);
+    currTeam = response;
+  });
+});
+
+$("#container").on("click", ".add-dream", function() {
+  const id = $(this)
+    .closest("div")
+    .data().id;
+  addDreamPlayer(currTeam.find(p => p.id === id));
+});
+
+const addDreamPlayer = function(player) {
+  $.post("/roster", player);
```

Comment:
Add call back function to receive data from the server or catch the error if something happened with the request